pandas/tests/arrays/test_integer.py

```diff
@@ -675,11 +673,23 @@ def test_reduce_to_float(op):
 
     # groupby
     result = getattr(df.groupby("A"), op)()
-    expected = pd.DataFrame({
-        "B": np.array([1.0, 3.0]),
-        "C": np.array([1, 3], dtype="float64")
-    }, index=pd.Index(['a', 'b'], name='A'))
-    tm.assert_frame_equal(result, expected)
+
+    assert result.dtypes['B'].name == 'float64'
+    assert result.dtypes['C'].name == 'Int64'
+
+
+@pytest.mark.parametrize('op', ['sum'])
```

Comment:
this is a duplicate of the above, no?

Reply:
correct me if I am wrong

Reply:
They seem to be doing the same thing (grouping by another column and reducing the Integer column). I think this one can be removed.